### PR TITLE
Add spinner to home feed

### PR DESCRIPTION
**Pull request by tylerhm**
# Changes
TSIA
IMG_2637

### CODE REVIEW

File: screens/home/HomeFeed.tsx [hunk 1]
```diff
@@ -207,6 +215,7 @@ const HomeFeed = () => {
           }
         />
       }
+      ListFooterComponent={renderSpinner}
       onEndReached={getNextPosts}
```
Comment (Henry-Graves) [inline screens/home/HomeFeed.tsx:219]: is there any kind of weird dead zone in which the spinner renders but onEndReached doesn't trigger? If not, lgtm

**Review by Henry-Graves — COMMENTED** [review]
see comment, will approve after

**Review by abrugal — CHANGES_REQUESTED** [review]
So I tested it and just like Henry said the onEndReached doesn't trigger. This was the issue I had before when trying to add the spinner. For some reason it messes up getting the onEndReached to get triggered.

**Comment by tylerhm** [comment]
Hmmm must be Android specific, and this behavior is already implemented on all other feeds so the bug just exist there too.

**Comment by tylerhm** [comment]
Are you able to put up a video of the bug? I can't repro.

**Comment by abrugal** [comment]
Actually it seems to be working. Thought it wasn't cause it took so long to load. Takes like 10 seconds or more to load new content. Is that the same for iOS?

**Review by abrugal — APPROVED** [review]
Works on Android. Took a while to load so I thought it wasn't fetching more posts

**Comment by tylerhm** [comment]
Yeah... network slow big sad

**Pull request merged**